Author: Henry-Nicolas Tourneur <debian@nilux.be>
Forwarded: not-needed
Description: This patch is removing a gresource since the svg file is stripped
 for licensing reason.  This resource is not used elsewhere in the application.
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/data/com.github.bilelmoussaoui.Authenticator.gresource.xml
+++ b/data/com.github.bilelmoussaoui.Authenticator.gresource.xml
@@ -3,9 +3,6 @@
   <gresource prefix="/com/github/bilelmoussaoui/Authenticator">
     <file alias="style.css">resources/gtk/style.css</file>
     <file>data.json</file>
-    <file alias="qrscanner-symbolic.svg">icons/hicolor/symbolic/actions/qrscanner-symbolic.svg</file>
-    <!-- Accounts logo fallback -->
-    <file alias="authenticator-symbolic.svg">icons/hicolor/symbolic/apps/com.github.bilelmoussaoui.Authenticator-symbolic.svg</file>
     <file alias="authenticator.svg">icons/hicolor/scalable/apps/com.github.bilelmoussaoui.Authenticator.svg</file>
 
     <!-- UI Files -->
--- a/data/icons/meson.build
+++ b/data/icons/meson.build
@@ -7,11 +7,5 @@ foreach theme : icon_themes
               install_dir: dest_scalable,
               rename: '@0@.svg'.format(application_id))
 
-  symbolic_dir = join_paths(theme, 'symbolic/apps/')
-  symbolic_icon = join_paths(symbolic_dir, meson.project_name() + '-symbolic.svg')
-  dest_symbolic = join_paths(get_option('prefix'), 'share/icons', symbolic_dir)
-  install_data(symbolic_icon,
-              install_dir: dest_symbolic,
-              rename: '@0@'.format(application_id) + '-symbolic.svg')
 endforeach
 
--- a/data/resources/gtk/ui/account_add.ui
+++ b/data/resources/gtk/ui/account_add.ui
@@ -56,7 +56,7 @@
               <object class="GtkImage" id="scan_img">
                 <property name="visible">True</property>
                 <property name="can_focus">False</property>
-                <property name="icon_name">qrscanner-symbolic</property>
+                <property name="icon_name">edit-select-all-symbolic</property>
               </object>
             </child>
           </object>
--- a/data/resources/gtk/ui/window.ui.in
+++ b/data/resources/gtk/ui/window.ui.in
@@ -260,7 +260,7 @@
                   <object class="GtkImage" id="qr_scanner_image">
                     <property name="visible">True</property>
                     <property name="can_focus">False</property>
-                    <property name="icon_name">qrscanner-symbolic</property>
+                    <property name="icon_name">edit-select-all-symbolic</property>
                     <style>
                       <class name="dim-label"/>
                     </style>
